github/provider.go

```diff
@@ -308,13 +318,21 @@ func providerConfigure(p *schema.Provider) schema.ConfigureFunc {
 		}
 		log.Printf("[DEBUG] Setting read_delay_ms to %d", readDelay)
 
+		parallelRequests := d.Get("parallel_requests").(bool)
+		isGithubDotCom, _ := regexp.MatchString("^https://api.github.com", baseURL)
+		if parallelRequests && isGithubDotCom {
+			return nil, fmt.Errorf("parallel_requests cannot be true when connecting to public github")
+		}
+		log.Printf("[DEBUG] Setting parallel_requests to %t", parallelRequests)
```

Comment:
What does this log print for a case of using github.com? It looks like it would print "Setting parallel_requests to false".

Reply:
yes, it is going to print that. Isn't it correct? I tried to keep the format consistent with other settings i.e.: `log.Printf("[DEBUG] Setting read_delay_ms to %d", readDelay)`

Reply:
You're right, I think I misread something, sorry for the confusion.